.env.example

```diff
@@ -1,3 +1,4 @@
+# Required
 # Subscription Keys
 BING_SEARCH_V7_WEB_SEARCH_SUBSCRIPTION_KEY
```

Comment:
I would suggest adding equal sign at the end `=`
```sh
BING_SEARCH_V7_WEB_SEARCH_SUBSCRIPTION_KEY=
....
```

Reply:
done. Since you all agree on this convention.
I've also search people's codes on GitHub and it does seem most popular.
Thanks everyone @kerolloz @Mohammed-Deifallah @nader-m-elhadedy